ledger/src/blockstore_processor.rs

```diff
@@ -3826,6 +3827,27 @@ pub mod tests {
             Err(TransactionError::AlreadyProcessed)
         );
 
+        // Make sure fees-only transactions still update the signature cache
```

Comment:
First process attempt will fail but still update status cache and the second attempt will be rejected since tx was already in status cache. Adding comments to make this clearer